### PR TITLE
Update client output and change log level

### PR DESCRIPTION
**Pull request by aarontp**
... Also adding sleep between task scheduling to smear out where tasks are executed.

### CODE REVIEW

File: turbinia/task_manager.py [hunk 1]
```diff
@@ -43,6 +43,7 @@
 log = logging.getLogger('turbinia')
 
 PSQ_TASK_TIMEOUT = 604800
```
Comment (berggren) [inline turbinia/task_manager.py]: Are these seconds?

I opt to make it clear in the constant name, e.g:
PSQ_TASK_TIMEOUT_SECONDS

File: turbinia/task_manager.py [hunk 2]
```diff
@@ -43,6 +43,7 @@
...
+PSQ_QUEUE_WAIT = 2
```
Comment (berggren) [inline turbinia/task_manager.py]: PSQ_QUEUE_WAIT_SECONDS

File: turbinia/task_manager.py (hunk 1 above)
Comment (aarontp) [reply turbinia/task_manager.py]: Yeah, seconds.  I'll update the name.

File: turbinia/task_manager.py (hunk 2 above)
Comment (aarontp) [reply turbinia/task_manager.py]: Done.

**Comment by aarontp** [comment]
Thanks for the review.  PTAL.

**Pull request merged**